synapse/http/site.py

```diff
@@ -126,13 +126,35 @@ def get_method(self) -> str:
             return self.method.decode("ascii")
         return method
 
+    def get_authenticated_entity(self) -> Optional[str]:
```

Comment:
This feels like it should potentially be two pieces of information in the structured logs? An authenticated entity and acted entity or something.

Reply:
Yeah, probably